src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
Agree, LTarget,Operand,RTarget looks better. Multiple constraints at the moment are separated automatically:

1,=,2,3,=,4
Makes two constraints: 1=2 and 3=4